yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/YotiConstants.java

```diff
@@ -13,11 +13,13 @@ private YotiConstants() {}
     public static final String AUTH_KEY_HEADER = "X-Yoti-Auth-Key";
     public static final String DIGEST_HEADER = "X-Yoti-Auth-Digest";
     public static final String YOTI_SDK_HEADER = "X-Yoti-SDK";
+    public static final String YOTI_SDK_VERSION_HEADER = YOTI_SDK_HEADER + "-Version";
```

Comment:
has this '-' as prefix a meaning?

Reply:
> has this '-' as prefix a meaning?

It looks like it's just a separator from `SDK` in `X-Yoti-SDK`

Reply:
Yes, it's just a separator for readability